pkg/cmd/roachtest/tests/costfuzz.go

```diff
@@ -40,7 +40,6 @@ func registerCostFuzz(r registry.Registry) {
 		Tags:            nil,
 		Cluster:         r.MakeClusterSpec(1),
 		Run:             runCostFuzz,
-		Skip:            "flaky test: https://github.com/cockroachdb/cockroach/issues/81717",
```

Comment:
Can we leave the skip in place for this PR?

Before we unskip costfuzz, I think we need to fix the failures mentioned in https://github.com/cockroachdb/cockroach/issues/81717.